src/ai/backend/common/events/dispatcher.py

```diff
@@ -672,10 +693,18 @@ async def broadcast_event(
         if source_override is not None:
             source = source_override
         args = base64.b64encode(msgpack.packb(event.serialize())).decode("ascii")
+
+        # Capture current request_id and other metadata
+        metadata = {}
+        request_id = current_request_id()
+        if request_id is not None:
+            metadata["request_id"] = request_id
+
         raw_event = {
             "name": event.event_name(),
             "source": source,
             "args": args,
+            "metadata": base64.b64encode(msgpack.packb(metadata)).decode("ascii"),
```

Comment:
Is the reason of encoding&decoding by base64 `orjson`?
Let's leave a comment then